package.json

```diff
@@ -0,0 +1,38 @@
+
+{
+    "name": "dynamites-ecomm-be",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+      "start:dev": "nodemon ./index.ts"
+    },
```

Comment:
Please add `build` script and normal `start` which doesn't use nodemon

Reply:
I  add both  build  and normal start  also  I use node instead  nodemon.